images/gh-gl-sync/SpackCIBridge.py

```diff
@@ -252,26 +282,23 @@ def get_open_refspecs(self, open_prs):
         """Return lists of refspecs for fetch and push given a list of open PRs."""
```

Comment:
Is this now just, e.g. "Return list of refspecs to push, given a list of open PRs"?

IIUC we no longer need to fetch refspecs for PRs because we had to do it already for each PR [here](https://github.com/spack/spack-infrastructure/pull/349/files#diff-822abeb2c9470f1fa0a4dd001219035b31166c1cf3a784f8248c38e93c3cdf2aR156)?

Reply:
Thanks your noticing this. Yes, your understanding is correct for why we no longer need to populate `fetch_refspecs` for open PRs.